nautobot/core/utils/migrations.py

```diff
@@ -33,3 +33,30 @@ def check_for_duplicates_with_natural_key_fields_in_migration(model_class, natur
             file=sys.stderr,
         )
         raise RuntimeError("Duplicate records must be manually resolved before migrating.")
+
+
+def update_object_change_ct_for_replaced_models(apps, new_app_model, replaced_apps_models, reverse_migration=False):
```

Comment:
Unit test looks reasonable - can we also add representative data to the `development/datasets/` DB dumps used in our CI migration tests as a regression test? From a quick look it looks like those dumps don't currently include any ObjectChange records at all.

Reply:
I've opened an issue against the private repo that the datasets were generated from, asking that it add such data.